### DISCUSSION TITLE
Idea: part-time master

### DISCUSSION THREAD

**Discussion by mhagdorn** [the-teachingRSE-project/RSE-Masters#41] [Ideas]
Following on from the question at HiRSE Seminar today, it occurred to me that it would be cool to have a part-time master. So that somebody working as an RSE can gain the qualification

**Comment by CaptainSifff** [comment]
Sounds like a good idea, but isn't this implicit in the Notion of the Master a little bit? If we take the idea of External institutions i can imagine offering sth. More lightweight than a Master so that you don't have to invest a full two years.

**Comment by juliandehne** [comment]
"Frage: Haben Sie eigentlich auch das Thema Nach-Ausbildung bereits als Wissenschaftliche Mitarbeiter tätiger Personen auch im Aufgabenbereich?
(oder mehr das Programm für Studierende)"

> Part-time study is something universities offer in any case. My suggestion is that professionals enrol in the master in that context. Organizing the courses and modules for part-time study should be the job of the Studiendekanat. 

Naturally, a 1-year Master would be more suitable in that case. So we should link the discussions with issue #2

**Comment by juliandehne** [comment]
Ob wir ein weiteres Profil anbieten, was berufsbegleitend oder 1-jährig ist, müssen wir dann unter dem Gesichtspunkt entscheiden, ob zu viele Varianten den Referenzaspekt verwässern.
